main/ertp/guide/create-non-fungible-tokens.md

```diff
@@ -60,20 +38,23 @@ Objects that represent valid tickets will have the following properties:
 
 Let's create the tickets in ERTP!
 The first step is to create JavaScript objects, each representing a ticket.
-Then, only [units](https://agoric.com/documentation/ertp/guide/units.html) can be minted, so let's create units from the JavaScript objects and then, let's mint the tickets!
+Then, only [amounts](./amounts) can be minted, so let's create amounts from the JavaScript objects and then, let's mint the tickets!
 
 ```js
 const startDateString = (new Date(2019, 11, 9, 20, 30)).toISOString();
 
-const ticketObjects = Array(1114).fill().map((_, i) => ({
-  seat: i+1,
-  show: 'The Sofa',
-  start: startDateString,
-}))
-
-const ticketUnits = ticketObjects.map(ticketExtent => balletTicketMint.getAssay().makeUnits(ticketExtent));
-
-const balletTicketPurses = ticketUnits.map(ticketUnit => balletTicketMint.mint(ticketUnit))
+const ticketDescriptionObjects = Array(1114).fill()
+  .map((_, i) => ({
+    seat: i + 1,
+    show: 'The Sofa',
+    start: startDateString,
+  }));
+
+const balletTicketPayments = ticketDescriptionObjects.map(
+  ticketDescription => balletTicketMint.mintPayment(
+    issuer.getAmountMath().make(harden([ticketDescription]))
+  )
+);
 ```
 
-For each ticket unit, we've created a [purse](https://agoric.com/documentation/ertp/api/purse.html) which contains the corresponding unit. These purses can later be transformed into [payments](https://agoric.com/documentation/ertp/api/payment.html) via a call to `balletTicketPurse.withdrawAll()`. This payment can then be bought and sold and used in smart contracts.
+For each ticket amount, a [payment](../api/payment.md) which contains the corresponding amount. These payments can then be used in smart contracts.
```

Comment:
For each ticket amount, a [payment](../api/payment.md) which contains the corresponding amount.
```suggestion
For each ticket amount, we've created a [payment](../api/payment.md) which contains the corresponding amount. These payments can then be used in smart contracts.
```